Jenkinsfile

```diff
@@ -143,25 +143,38 @@ def stepGetRiotUrlAndCommit() {
  * @param ver           Type of version to get {master, pull, current, submodule}
  * @param repo_owner    Repoisitory owner
  * @param repo_name     Repoisitory name
- * @param pull          Optional pull request number
+ * @param ver_arg       Optional pull request number or relase tag
  *
  * @return (url, commit)
  */
-def getUrlAndCommitFromParams(ver, repo_owner, repo_name, pull = 0) {
+def getUrlAndCommitFromParams(ver, repo_owner, repo_name, ver_arg = 0) {
     if (ver == 'master') {
+        sh script: "echo ${repo_name} using master",
+            label: "${repo_name} using master"
         return getUrlAndCommitFromDefault(repo_owner, repo_name)
     }
     else if (ver == 'pull') {
-        assert pull != '0' : "Selected pull of ${repo_name} but no pull entered"
-        return getUrlAndCommitFromPr(repo_owner, repo_name, pull)
+        sh script: "echo ${repo_name} using pull request ${ver_arg}",
+            label: "echo ${repo_name} using pull request ${ver_arg}"
+        assert ver_arg != '0' : "Selected pull of ${repo_name} but no pull entered"
+        return getUrlAndCommitFromPr(repo_owner, repo_name, ver_arg)
     }
     else if (ver == 'current') {
+        sh script: "echo ${repo_name} using current",
+            label: "echo ${repo_name} using current"
         return [env.GIT_URL.take(env.GIT_URL.lastIndexOf('.')),
                 env.GIT_COMMIT]
     }
     else if (ver == 'submodule') {
+        sh script: "echo ${repo_name} using submodule",
+            label: "echo ${repo_name} using submodule"
         return getUrlAndCommitOfSubmodule("RIOT-OS", "RobotFW-Tests")
     }
+    else if (ver == 'release') {
+        sh script: "echo ${repo_name} using release ${ver_arg}",
+            label: "echo ${repo_name} using release ${ver_arg}"
```

Comment:
```suggestion
            label: "echo ${repo_name} using release ${ver_arg}"
        assert ver_arg != '0' : "Selected release of ${repo_name} but no release version entered"
```

Reply:
I don't think it is worth rerunning tests for this change, it should fail if nothing is found (the API and parsing would crash it)